website/src/theme/MDXComponents/index.tsx

```diff
@@ -80,6 +80,32 @@ export default {
     <blockquote className={styles.blockquote} {...props} />
   ),
 
+  DartPad: ({ path, branch }: { path: string; branch?: string }): JSX.Element => {
+    return (
+      <div
+        style={{
+          marginBottom: '2rem',
+          position: 'relative',
+          paddingTop: '56.25%',
+        }}
+      >
+        <iframe
+          src={`https://dartpad.dev/embed-flutter.html?theme=dark&run=true&split=70&null_safety=true&gh_owner=FirebaseExtended&gh_repo=flutterfire&gh_ref=${
```

Comment:
Any way to set `theme=` here to match the value of the site theme toggle?

Reply:
Yeah good idea, done.